### PR TITLE
Release/0.5.0

### PR DESCRIPTION
**Pull request by adatzer**
This release includes the new feature contributed by @spinx in #16 !

### CODE REVIEW

**Review by igneel64 — APPROVED** [review]

File: template.tpl [hunk 1]
```diff
@@ -794,46 +832,51 @@ const unixMillisToISO = (unixMillis) => {
...
-        '_'
-      ),
-      '/',
-      '_'
-    );
+    const rexp = createRegex('[./]', 'g');
+    let fixed = schema
+      .replace('iglu:', '')
+      .replace('jsonschema/', '')
+      .replace(rexp, (match) => {
```
Comment (jethron) [inline template.tpl]: Can't this just be `.replace(rexp, '_')`?

File: template.tpl [hunk 2]
```diff
@@ -257,13 +257,47 @@ ___TEMPLATE_PARAMETERS___
...
+            "displayName": "Self Describing Event Location",
+            "macrosInSelect": false,
+            "selectItems": [
+              {
+                "value": "object",
+                "displayValue": "Schema name object on Braze Event"
+              },
+              {
+                "value": "root",
+                "displayValue": "SDE Event Properties added to Braze Event Root"
```
Comment (jethron) [inline template.tpl]: These are pretty wordy for a select box. Could it just be like "Nested Object"  or "Merge to Root"?

**Review by jethron — APPROVED** [review]
LGTM!

File: template.tpl (hunk 1 above)
Comment (adatzer) [reply template.tpl]: Much better that way, thank you for spotting! Changed.

File: template.tpl (hunk 2 above)
Comment (adatzer) [reply template.tpl]: Changed them to:

- "Nest under schema name"
- "Merge to root level"

Maybe in the future we'd like to add an option to "Nest under custom path", that is why i specified the "under schema name" for the default option.

Also tweaked the help text to:
"Indicate the location where Snowplow Self Describing event properties should be added under Braze event properties.".

Do these sound ok? What do you think?

Comment (jethron) [reply template.tpl]: Sounds good, thanks!

**Pull request merged**